doc/release-notes/rl-2511.section.md

```diff
@@ -33,4 +33,6 @@
 
 ### Additions and Improvements {#sec-nixpkgs-release-25.11-lib-additions-improvements}
 
-- Create the first release note entry in this section!
+- Neovim Wrapper: Added support for the `vim.o.exrc` option, as well as the `VIMINIT` environment variable, and sourcing of `sysinit.vim`.
+
```

Comment:
Did you check this renders correctly?

Looking at rl2505, it may be more common to format this as a nested list item.

Reply:
LGTM: 
![image](https://github.com/user-attachments/assets/b6fc4380-a9d6-44e2-ba03-268c16414f94)